Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -455,6 +455,7 @@ void CConfigMain::InitializeGUITooltips()
 {
 	// General - Basic
 	CPUThread->SetToolTip(_("This splits the Video and CPU threads, so they can be run on separate cores.\nCauses major speed improvements on PCs with more than one core, but can also cause occasional crashes/glitches."));
```

Comment:
How about Processor and Video? It's consistent and easy for the average user to understand.